src/components/Apps/App.js

```diff
@@ -5,13 +5,15 @@ import PasswordResetPage from 'pages/PasswordReset';
 import PageNotFound from 'pages/Error';
 
 import 'assets/MainStyle.scss';
+import Comment from 'components/Comment';
 
 class App extends Component {
   render() {
     return (
       <Router>
         <Switch>
           <Route exact path="/" render={props => <LandingPage {...props} />} />
+          <Route path="/comments" render={props => <Comment {...props} articleSlug="javascript-code" />} />
```

Comment:
Nice Work, @llwasampijja , I see you are hard coding the slug, you could pass it from the URL and pick it like this: `<Route path="/comments/:slug" render={props => <Comment {...props} />} />` then from your component you can then pick from 
`const { match } = this.props;
    if (match) {
      const { params: { slug } } = match;
    }`

Reply:
@nadralia this is just for testing, I will be picking the slug and comment id from the comment component